visualize_vispy_lines.py

```diff
@@ -105,6 +107,28 @@ def visualize(buf, get_idx_fn, update_callback_fn, acquisition_fun=None):
     xax = scene.AxisWidget(orientation='bottom', axis_label=f"Time in (1/{BIN_SIZE})s", tick_label_margin=15)
     grid.add_widget(xax, GRID_ROWS, 1, col_span=GRID_COLS)
     xax.link_view(view)
+    
+    corr_bins = np.logspace(np.log(1/ACQUISITION_RATE)/np.log(10), 0, 10)
```

Comment:
For the UI part I'm waiting for your discussion with Gabriel to resolve to understand whether we need/want it.

Reply:
In my latest update I give command line options -c and -a to include cross and auto correlation, if none, then it looks like the previous.
Let's see what Gabriel thinks